source/ht_inireader.cpp

```diff
@@ -62,6 +62,39 @@ namespace Hatchit {
             return key;
         }
 
+        char* INIReader::StreamReader(char* str, int len, void* stream)
```

Comment:
Since this function doesn't access any private members of INIReader, we could make this a static local function instead of a private static function of INIReader.  Once again, gray area of our standards right now, so I think either way is fine.